pom.xml

```diff
@@ -15,8 +15,8 @@
         <maven>3.0.5</maven>
     </prerequisites>
 
-    <groupId>com.googlecode.maven-java-formatter-plugin</groupId>
-    <artifactId>maven-java-formatter-plugin</artifactId>
+    <groupId>com.github.java-formatter-maven-plugin</groupId>
```

Comment:
Now that the project has been transferred, if your intentions are to release under the net.revelc groupId to maven central, this PR should be updated to reflect the new groupId (should be `net.revelc.code`) and artifactId (should be `formatter-maven-plugin`).